docs/user-guide/cli-cicsplugin.md

```diff
@@ -41,27 +36,58 @@ Use one of the following methods to install or update the plug-in:
 
 ## Creating a user profile
 
-You create a cics profile to avoid entering your connection details each time that you issue a command. You can create multiple profiles and switch between them as needed. Use one of the following methods to create a profile:
-- **Create plug-in profiles using a configuration file:** Specify your profile and connection details in the `zowe.config.json` configuration file.
-- **Create plug-in profiles using a command:** Issue the `zowe profiles create` command to create the profile.
-We recommend that you create profiles using the configuration file. We do not recommend using profile commands because we are removing them from a future major release.
+After you install the plug-in, create a CICS profile to avoid entering your connection details each time that you issue a command. You can create multiple profiles and switch between them as needed.
+
+Specify your plug-in profile and connection details in the `zowe.config.json` configuration file.
 
 ### Creating plug-in profiles using a configuration file
 
-When you issue various `zowe config` commands, such as `init`, `auto-init`, and `convert-profiles`, they create a `zowe.config.json` configuration file. When you install the CICS plug-in, the commands create an entry for a cics profile in your `zowe.config.json` file.
+When you issue various `zowe config` commands, such as `init`, `auto-init`, and `convert-profiles`, they create a `zowe.config.json` configuration file. When you install the CICS plug-in and then issue a `zowe config` command **[correct?]**, the command creates an entry for a CICS profile in your `zowe.config.json` file.
```

Comment:
@adam-wolfe - please respond

Reply:
This is true, however something about the order of the sentences is a bit weird.

It might be simpler to say something like "If you have the CICS plug-in installed and issue the `zowe config init`, `zowe config auto-init`, or `zowe config convert-profiles` command, the command creates an entry for a CICS profile in your `zowe.config.json` file.